### PR TITLE
Feature/vulnanalysis complete notification

### PR DESCRIPTION
**Pull request by mehab**
Added changes on hyades services side to address vuln analysis complete notification. Addresses issue: https://github.com/DependencyTrack/hyades/issues/573

### CODE REVIEW

File: scripts/create-topics.sh [hunk 1]
```diff
@@ -40,6 +40,8 @@ notification_topics=(
 for topic_name in "${notification_topics[@]}"; do
   create_topic "$topic_name" "${NOTIFICATION_TOPICS_PARTITIONS:-3}" "retention.ms=${NOTIFICATION_TOPICS_RETENTION_MS:-43200000}"
 done
+create_topic "${API_TOPIC_PREFIX:-}dtrack.notification.project-vuln-analysis-complete" "${NOTIFICATION_TOPICS_PARTITIONS:-3}" "cleanup.policy=compact" "retention.ms=${NOTIFICATION_TOPICS_RETENTION_MS:-43200000}"
```
Comment (sahibamittal) [inline scripts/create-topics.sh:43]: Have we also submitted the topic creation in MetaManager?

Comment (sahibamittal) [reply scripts/create-topics.sh:43]: Need to add same in TOPICS.md

Comment (mehab) [reply scripts/create-topics.sh:43]: will also submit to metamanager

File: TOPICS.md [hunk 2]
```diff
@@ -39,6 +40,8 @@
 | `hyades-vulnerability-analyzer-snyk-retry-store-changelog`<sup>1E</sup>                           | 3          | `cleanup.policy=compact`<br/>`segment.bytes=67108864`<br/>`max.compaction.lag.ms=0` |
 
 *<sup>1X</sup> The topic is subject to [co-partitioning requirements](#co-partitioning-requirements)*  
+*<sup>1D</sup> To use this notification, the PROJECT_VULN_ANALYSIS_COMPLETE group needs to be manually added through api call to the notify on condition
```
Comment (nscuro) [inline TOPICS.md:43]: Please use `<sup>3</sup>` instead. `1` is supposed to only be used for highlighting co-partitioning requirements.

Also please ensure the formatting is correct when rendered, at the moment it's kind of broken:

image

File: TOPICS.md [hunk 3]
```diff
@@ -39,6 +40,8 @@
...
+The put request to be executed is [here](https://github.com/DependencyTrack/hyades-apiserver/blob/main/src/main/java/org/dependencytrack/resources/v1/NotificationRuleResource.java#L100) *
```
Comment (nscuro) [inline TOPICS.md:44]: Could you raise an issue for adding this to the frontend, so we don't forget about it?

Comment (mehab) [reply TOPICS.md:44]: okay

**Review by sahibamittal — APPROVED** [review]
lgtm 👍🏼

**Pull request merged**